native/native_main.py

```diff
@@ -14,43 +15,527 @@
 import unicodedata
 
 DEBUG = False
-VERSION = "0.1.6"
+VERSION = "0.1.7"
+DEFAULT_UNIX_SHELL = "/bin/sh"
+WIN_NATIVE_DIRNAME = ".tridactyl"
+PRE_RESTART_HOOK_PREFIX = "pre_restart_hook-"
 
```

Comment:
The native messenger is getting quite big. Perhaps we could split it in three, have a "common" base, a file with windows utilites and another with unix utilities and have the installer download and concatenate two files and concatenating depending on the OS?

Reply:
1. I'm fine with splitting, cleaning up _etc._ the source file(s)

2. I think we need to clearly define how we'd ship the final `native_main.py` to the user, and also evaluate how we develop, debug _etc._